src/devices/cpu/diablo/diablo1300.cpp

```diff
@@ -93,6 +113,7 @@ void diablo1300_cpu_device::device_start()
 	m_program = &space(AS_PROGRAM);
 	m_data    = &space(AS_DATA);
 	m_cache  = m_program->cache<1, -1, ENDIANNESS_LITTLE>();
+	m_table = (uint8_t*) machine().root_device().memregion("trom")->base();
```

Comment:
This tag should be made either relative to the device or a configuration parameter.

Reply:
Yep, I thought so, just need to verify that it works